scipy/stats/_continuous_distns.py

```diff
@@ -21,7 +21,6 @@
 import scipy.special._ufuncs as scu
 from scipy._lib._util import _lazyselect, _lazywhere
 from . import _stats
-from ._rvs_sampling import rvs_ratio_uniforms
```

Comment:
This and `from cpython.pycapsule cimport PyCapsule_IsValid, PyCapsule_GetPointer` below don't look related. Just some things you wanted to get done? If so, no problem. Just wanted to make sure they're intentional.

Reply:
Oops, this is related. How about they PyCapsule stuff, though?

Reply:
The pycapsule import can be removed